packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.ts

```diff
@@ -290,6 +350,24 @@ export class RipgrepSearchInWorkspaceServer implements SearchInWorkspaceServer {
         return Promise.resolve(searchId);
     }
 
+    private resolveSearchPaths(rootPaths: string[], opts: SearchInWorkspaceOptions | undefined): string[] {
```

Comment:
`protected` for the same reason as above.

Reply:
Done, thanks!